src/wp-admin/plugin-install.php

```diff
@@ -24,6 +24,17 @@
 	exit;
 }
 
+if ( isset( $_GET['activate'] ) ) {
+	wp_admin_notice(
+		__( 'Plugin activated.' ),
+		array(
+			'id'                 => 'message',
+			'additional_classes' => array( 'updated' ),
+			'dismissible'        => true,
+		)
+	);
+}
```

Comment:
A Warning for headers already sent happens with some plugins (such as bbPress, Custom Twitter Feeds, etc) and inconsistently with many others.

In testing with @costdev, found that this new admin notice is the likely suspect. Commenting this out and/or moving it to the end of the file (before the footer) resolves the warning.

Reply:
Relocated in [21e4899](https://github.com/WordPress/wordpress-develop/pull/6732/commits/21e4899565c9ddb1a493d26bb055f1eaa6c8b221).